src/agent/src/main.cpp

```diff
@@ -34,6 +32,20 @@ int main(int argc, char* argv[])
         {
             StopAgent();
         }
+        else if (cmdParser.OptionExists("install"))
+        {
+            InstallService();
+            return 0;
```

Comment:
Some comment for `RemoveService`, is there no way to tell if any of them failed? We always return 0, which is a success return value, if we can tell that installing or removing the service failed, we could return a non zero value.

Reply:
Fixed!